lib/src/asset_state.cc

```diff
@@ -76,6 +77,17 @@ AssetState::Asset::Asset(fty_proto_t* message)
         }
     }
     zhash_destroy(&ext);
+
```

Comment:
Same for endpoint.2 ?

Reply:
No... endpoint.2 is optional, depending the device card and the user needs...